connipc_windows.go

```diff
@@ -0,0 +1,87 @@
+// +build windows
```

Comment:
usually, this comes up after the license blurb.

Reply:
I didn't know that worked -- I've used this scheme in my other projects with the same placement.  Thinking about it though, I think it's actually better having this where it is -- it immediately calls out the platform(s) where this relevant (or not).  Its a small enough thing (one line only) that I don't think it dilutes the license blob in many meaningful way where it is.

Reply:
Conversely, our license blurb is a little longer (shorter than the BSD blurb in the go runtime for example) -- so sticking this below would easily lead it to being "lost" in the confusion. I'm also not sure if go needs this in the first "n" lines, or just before the first non-whitespace/non-comment line, or anywhere in the file.

At any rate, I plan to leave it where it is -- at least unless it triggers some warning from go vet or something like that.  (Pretty sure it doesn't.)

Reply:
I'm seeing some failures with containers and in the test suite at AppVeyor:

The problem is the underlying named pipe library -- the following fix should address it:

https://github.com/Microsoft/go-winio/pull/75

This is also described here:

https://github.com/Microsoft/go-winio/issues/67